libtransmission/quark.cc

```diff
@@ -283,6 +283,7 @@ auto constexpr MyStatic = std::array<std::string_view, 401>{ ""sv,
                                                              "recent-relocate-dir-4"sv,
                                                              "recheckProgress"sv,
                                                              "remote-session-enabled"sv,
+                                                             "remote-session-https"sv,
```

Comment:
Since this adds a new string constant, the count at the up at the top needs to be bumped from `401` to `402`

Reply:
funny: i did do that in the original patch, but then rebased on latest and git lost that change because it was already there... tricky!